src/test-utils.tsx

```diff
@@ -21,19 +21,21 @@ import { ConfigProvider } from 'antd'
 import { ReactElement } from 'react'
 
 import { ThemeContext, defaultTheme } from './components/ThemeProvider'
+import { ANT_PREFIX } from './components/ThemeProvider/Ant'
 
 /**
  * Wraps a React element with a series of providers for testing purposes.
  *
  * @remarks It adds the default theme to components
+ * @remarks It adds the default Ant class prefix
  * @remarks It removes the css-do-not-override classes from Ant Design components
  *
  * @param {ReactElement} children - The React element to be wrapped.
  * @returns {ReactElement} The wrapped React element.
  */
 const TestWrapper = ({ children }: {children: ReactElement}): ReactElement => (
```

Comment:
okay, the goal is to have a single wrapper to handle everything you need for the test mock. In this case we can also keep it like this

Reply:
Do we add a how to use somewhere with an example in jest? For example create a custom renderComponent that has him as wrapper and then use him for component renderings

Reply:
Even if this wrapper almost replicates the ThemeProvider structure at the moment, in my opinion, that is not its only purpose. In the future, it might include additional configurations only used in tests, so I would prefer if the name of this wrapper would also express this. 

Inside the DS context, I think TestWrapper is indeed the correct name since it provides all the necessary configurations to run tests for every component. 
I agree that when used outside the DS context, also in combination with other wrappers, it might be confusing.

Maybe something like ComponentsTestWrapper? Also, maybe the best solution is to rename it according to its usage:
```
import {testUtils as dsTestUtils} from @mia-platform/console-design-system-react

const {TestWrapper as ComponnetsWrapper} = dsTestUtils
```

Reply:
> Do we add a "how to use somewhere" with an example in jest? For example, create a custom render component that has him as a wrapper and then use him for component renderings
> 
> Like this? `export const renderComponent = Component => render(Component, {wrapper: TestWrapper})`

I think also this can be addressed in #321